Volume/View Models/HomeViewModel.swift

```diff
@@ -0,0 +1,216 @@
+//
+//  HomeViewModel.swift
+//  Volume
+//
+//  Created by Hanzheng Li on 10/26/22.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+
+extension HomeView {
+    @MainActor class ViewModel: ObservableObject {
+        typealias ArticlesResultPublisher = Publishers.Map<OperationPublisher<GetArticlesByPublicationSlugsQuery.Data>, [ArticleFields]>
+
+        private struct Constants {
```

Comment:
Move to extension if this gets longer

Reply:
Or maybe now too if you want to be proactive

Reply:
I wanted to do this, but you can't write extensions within embedded structs (this one's embedded within `HomeView`) ;(